integration/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -1816,9 +1818,9 @@ class OntologyV2R2RSpec extends R2RSpec {
 
           val graph        = responseJsonDoc.body.requireArray("@graph").value.map(_.asInstanceOf[JsonLDObject])
           val nothingValue = graph.filter(_.requireString("@id") == hasOtherNothingValueIri).head
-          val isEditableMaybe = UnsafeZioRun.runOrThrow(
-            nothingValue.getBoolean(KnoraApiV2Complex.IsEditable).mapError(BadRequestException(_))
-          )
+          val isEditableMaybe =
+            nothingValue.getBoolean(KnoraApiV2Complex.IsEditable).fold(throw BadRequestException(_), identity())
```

Comment:
Back to throwing? This looks like in entire file.

Reply:
`UnsafeZioRun.runOrThrow` would have thrown the `BadRequestException` anyway. What to you propose to do instead in the test?